src/test/java/teammates/test/pageobjects/InstructorFeedbackEditPage.java

```diff
@@ -713,6 +721,25 @@ public boolean isAllFeedbackPathOptionsEnabled() {
         return true;
     }
     
+    public boolean isAllRecipientOptionsDisplayed(int questionNumber) {
+        List<WebElement> recipientOptions =
+                browser.driver.findElements(By.cssSelector("#recipienttype-" + questionNumber + " option"));
+        for (WebElement recipientOption : recipientOptions) {
+            if ("none".equals(recipientOption.getCssValue("display"))) {
+                return false;
+            }
+        }
+        return true;
+    }
+    
+    public boolean isRecipientOptionDisplayed(FeedbackParticipantType recipientType, int questionNumber) {
+        WebElement recipientOption =
+                browser.driver.findElement(
+                        By.cssSelector("#recipienttype-" + questionNumber
+                                       + " option[value='" + recipientType + "']"));
+        return "block".equals(recipientOption.getCssValue("display"));
```

Comment:
Displayed elements can have `display` values of block, flex, inline, etc. (although it might not be the case here), so it should be better to check for displayed with `!"none".equals(...)` instead.